pkg/apis/newrelic/v1alpha1/monitor_types.go

```diff
@@ -215,21 +221,24 @@ func (s *Monitor) GetID() string {
 
 // Update object in newrelic
 func (s *Monitor) Update(ctx context.Context) error {
-	//TODO Updates are failing
-	// rsp, err := clientSythetics.SyntheticsMonitors.Update(ctx, s.toNewRelic(), s.Status.ID)
-	// err = handleError(rsp, err)
-	// if err != nil {
-	// 	s.Status.Info = err.Error()
-	// 	return err
-	// }
-
-	err := s.updateCondition(ctx)
-	if err != nil {
-		s.Status.Info = err.Error()
-		return err
-	}
+	if s.Spec.ManageUpdates == nil || (s.Spec.ManageUpdates != nil && *(s.Spec.ManageUpdates)) {
+		rsp, err := clientSythetics.SyntheticsMonitors.Update(ctx, s.toNewRelic(), s.Status.ID)
+		err = handleError(rsp, err)
+		if err != nil {
+			s.Status.Info = err.Error()
+			return err
+		}
 
-	s.Status.Hash = s.Spec.GetSum()
+		err = s.updateCondition(ctx)
+		if err != nil {
+			s.Status.Info = err.Error()
+			return err
+		}
+
+		s.Status.Hash = s.Spec.GetSum()
+	} else {
+		log.Warn(fmt.Sprintf("Skipping update for %s", s.Signature()))
```

Comment:
the signature should not be needed.  If you fetch the logger from the ctx.

Reply:
Done.